creativecloud/scripts/utils.js

```diff
@@ -58,13 +58,23 @@ function getDecorateAreaFn() {
     if (lcpImg) lcpImgSet = true;
   };
 
+  function replaceDotMedia(area = document) {
```

Comment:
For the gnav, we also replace some media hrefs in case it's a [shared/centralised](https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/utilities/utilities.js#L109) nav. This is something we should keep in mind and test.

I created a test page for [this scenario here](https://mwpw-142577--cc--adobecom.hlx.page/drafts/osahin/test-gnav?milolibs=mwpw-144020--milo--mokimo)

Reply:
As far as I can tell, we are not running into any issues 
![image](https://github.com/adobecom/cc/assets/39759830/4f634508-edd3-4d75-9be0-f6666aed875e)